schedule/yast/encryption/cryptlvm_iscsi.yaml

```diff
@@ -11,40 +11,34 @@ vars:
   NICTYPE: user
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/access_beta_distribution
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/iscsi_configuration
-  - installation/registration/register_via_scc
-  - installation/module_registration/register_module_desktop
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_SLES_with_GNOME
-  - installation/partitioning/select_guided_setup
-  - installation/partitioning/guided_setup/select_disks
-  - installation/partitioning/guided_setup/encrypt_lvm_simple_pwd
-  - installation/partitioning/guided_setup/accept_default_fs_options
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_default_target
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/grub_test
-  - installation/boot_encrypt
-  - installation/first_boot
-  - console/system_prepare
-  - console/hostname
-  - console/force_scheduled_tasks
-  - installation/validation/ibft
-  - console/validate_lvm
-  - console/validate_encrypt
+  registration:
+    - installation/iscsi_configuration
```

Comment:
this is not related with registration, should overwrite `disk_activation` in default file.

Reply:
I added `disk_activation` to the `default_x86_64.yaml` file in PR#16218.

Reply:
OK